sympy/physics/secondquant.py

```diff
@@ -1021,11 +1069,17 @@ def up(self, i):
         """
         Performs the action of a creation operator.
 
+        Explanation
+        ===========
+
         If below fermi we try to remove a hole,
-        if above fermi we try to create a particle.
+        If above fermi we try to create a particle.
```

Comment:
I think this doesn't needs to capitalise as there is a comma right before it.

Reply:
Changed.